code/WorkInProgress/kilakk/fax.dm

```diff
@@ -139,8 +139,9 @@ var/list/alldepartments = list("Central Command")
 
 	if(href_list["remove"])
 		if(tofax)
-			tofax.loc = usr.loc
-			usr.put_in_hands(tofax)
+			tofax.forceMove(loc)
```

Comment:
Because putting it to usr's location was what caused the teleporting paper to begin with, this way it goes to the fax machine instead of the person using it unless they're close enough/able to pick it up as well.